packages/blocks/calculation/README.md

```diff
@@ -0,0 +1,13 @@
+# Calculation
+
```

Comment:
Let’s remove this H1 as in other blocks (this improves README display in block hub)